00--Ready-Player-One/README.md

```diff
@@ -1,24 +1,81 @@
+
+### You've Got Issues
+
+[Create an Issue in the class repo (this one)](../issues/new) and name it `00 -- YOUR NAME`, where `YOUR NAME` is, well... your name. Your real, full name. If you copy and paste the checklist from the **Tasks** sections below into the **Description** of your issue _as is_, you should end up with a bunch of checkboxes thanks to [the way that GitHub interprets text](https://guides.github.com/features/mastering-markdown/). As you complete parts of the assignment, check off the corresponding checkboxes and watch your progress bar fill! Read [Mastering Issues](https://guides.github.com/features/issues/) for more information on creating issues in GitHub.
+
+_This task list should be complete by the first day of class!_
+
+### Send all my GitHub to you...
+
+Finally, if you haven't guessed it yet, GitHub is a tool we'll use very heavily in the class and is highly respected in the industry. Learn all you can about `git` -- the command -- and GitHub! There are [some helpful Guides on GitHub](http://guides.github.com) to help you get started, but start with [Hello World](https://guides.github.com/activities/hello-world/) and [Fork-and-Spoon](https://guides.github.com/activities/forking/)
+
+## Command Line Training
+
+In this class, we're going to use the Command Line Interface (CLI) _a lot_. Start with [_An Intro to the OS X Command Line_](http://eddywashere.com/blog/an-intro-to-the-os-x-command-line/) and [_Learn the Command Line_ on CodeCademy](https://www.codecademy.com/courses/learn-the-command-line/). If you're looking for more of a challenge, you can try the [_CLI Crash Course_ by Zed Shaw](http://cli.learncodethehardway.org/book/).
+
+## INSTALL ALL THE THINGS!
+
+Throughout the course, we'll be using a lot of tools, some of them on day 1. _Please have these installed on your Mac before we get started on Monday!_ If you have trouble, please put a comment on your WIP Issue and notify us via email. _Don't stop!_
+
+### Prezto Change-oh!
+
+One of the tools we'll frequently use in class is Prezto. To test your Google Fu and GitHub prowess: find, fork, and install Prezto on GitHub. The install instructions are a little tricky, so pay careful attention. If you get stuck, please leave a comment on your issue and move on.
+
+## Take a Moment to Reflect
+Please answer the following two questions in short essay form. Try to keep each answer between 250 - 500 words. Don't worry too much about grammar or spelling -- use this time to reflect on yourself and your upcoming adventure!  Please submit your answers as a comment on your WIP Issue. 
+
+1. What difficulties do you predict that you will encounter during our course? How have you reacted to similar situations in the past? How do you plan on reacting to these situations during the class?
+
+2. Which part of the prework did you like the MOST? Which part of the prework did you like the LEAST? If you found any of the prework confusing, please explain.
```

Comment:
If you ditch the extra space between these and make the lines both start with `1.`, GitHub / Markdown will make them a numbered list, e.g.

1. First Thing
1. Second Thing